tests/e2e/e2e_setup_test.go

```diff
@@ -290,7 +334,7 @@ func (s *IntegrationTestSuite) runValidators(chainConfig *chainConfig, portOffse
 func (s *IntegrationTestSuite) runIBCRelayer(chainA *chainConfig, chainB *chainConfig) {
 	s.T().Log("starting Hermes relayer container...")
 
-	tmpDir, err := ioutil.TempDir("", "osmosis-e2e-testnet-hermes-")
+	tmpDir, err := os.MkdirTemp("", "osmosis-e2e-testnet-hermes-")
```

Comment:
Why? I remember seeing an issue in `cosmos-sdk` that one of these is causing trouble in CI. Is that the reason for this change?

Reply:
As of go 1.17, ioutil.TempDir just calls os.MkdirTemp. The change was made originally to see if it would fix the permissions issue I was having (it didn't and I solved it another way), but I didn't revert this change because it seems like we might as well call the function directly instead of inadvertently.